stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/application/ChatInitializer.kt

```diff
@@ -58,6 +60,16 @@ class ChatInitializer(private val context: Context) {
             .withPlugin(offlinePlugin)
             .build()
 
+        InitializationCoordinator.getOrCreate().addUserConnectedListener {
```

Comment:
Do we need to do the same change in compose sample?

Reply:
At the end, there's no need for this code.